src/libraries/Microsoft.PowerFx.Core/Public/FunctionInfo.cs

```diff
@@ -39,11 +44,98 @@ internal FunctionInfo(TexlFunction fnc)
         /// <summary>
         /// Get a short description of the function.  
         /// </summary>
+        [Obsolete("Use GetDescription() which takes a locale")]
         public string Description => _fnc.Description;
 
+        /// <summary>
+        /// Get a short description of the function.
+        /// </summary>
+        /// <param name="culture"></param>
+        /// <returns></returns>
+        public string GetDescription(CultureInfo culture = null)
+        {
+            culture ??= CultureInfo.InvariantCulture;
```

Comment:
>CultureInfo.InvariantCulture [](http://example.com/codeflow?start=24&length=28)

Shouldn't we get the ParseCultureInfo via FunctionInfo constructor instead of this default?